### ISSUE TITLE
running ./app.py after clean install displays an error

### ISSUE THREAD

**Issue by surfac3** [21isenough/LightningATM#96]
When ATM is setup and `./app.py` is run for the first time after `config.ini` is setup, the console returns: 
![telegram-cloud-photo-size-4-5832649915812135563-y](https://github.com/21isenough/LightningATM/assets/89400663/62785b6d-de4f-411a-a6b7-76d7e22f362f)

Display test runs with positive results. I am running `waveshare2in13v3` display

Steps to reproduce
a) clean SD image etched
b) go trough update and upgrade
c) runn ./app.py to setup config.ini
d) setup config.ini
e) run `./app.py` once more and it wont run

I am not sure why console displays blink error.

**Comment by surfac3** [comment]
Could it be somehow connected to this? 

![telegram-cloud-photo-size-4-5832649915812135573-y](https://github.com/21isenough/LightningATM/assets/89400663/d5d7059d-837b-4dea-a0fc-efe0f0af0b36)

its the output of `sudo apt update && sudo apt upgrade`

**Comment by openoms** [comment]
that update error is because of the ancient image.
Python needs to be min 3.6 for that printf line to work.

Check your:
```
python --version
```

I recommend using a new image based on Debian Bullseye or Buster at least.

**Comment by surfac3** [comment]
Right that very well may be...however i was able to find a workaround and that was using this:
https://github.com/AxelHamburch/LightningATM/blob/doc_lnbits_funding/docs/guide/we_need_your_help.md

Which is when lnbits was just implemented and this worked for me and ATM is functional

It seems to me that the steps in installation described in the master Readme arent leading to fuctioning ATM.

**Comment by AxelHamburch** [comment]
This branch is obsolete. The Blink wallet came after the implementation of the LNbits wallet. I would only use the current version.
![Screenshot_20240416-172510_Chrome](https://github.com/21isenough/LightningATM/assets/84231978/40a71e98-f163-4157-97f9-6e5771856ef6)

**Comment by surfac3** [comment]
Yes it is obselete. However it works with the old image which is described in the readme to use when setting up. So either there needs to be a new image tested and change the readme to the new image or the instructions need to change to lets say this older PR which works with the old image - sadly this would negate the Blink implementation so id think its easier to use the new image. I can test out the new image when i will build new ATM
